packages/bento-design-system/src/Stepper/createStepper.tsx

```diff
@@ -23,21 +24,13 @@ export function createStepper(config: StepperConfig) {
         {steps.map(({ label }, index) => {
           const status =
             index < currentStep ? "done" : index === currentStep ? "inProgress" : "todo";
-          return <Step label={label} index={index} status={status} key={label} />;
+          return <Step label={label} index={index} status={status} key={index} />;
```

Comment:
I've done a similar refactor for ListItem, I would suggest to include an optional `key` property to StepProps so that once can customize it in those cases where using index is problematic

Reply:
makes sense